Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -453,9 +444,9 @@ GLubyte* SoftwareGLContext::gl_get_string(GLenum name)
 
     switch (name) {
     case GL_VENDOR:
-        return reinterpret_cast<GLubyte*>(const_cast<char*>(m_device_info.vendor_name.characters()));
+        return reinterpret_cast<GLubyte*>(const_cast<char*>(m_rasterizer.info.vendor_name.characters_without_null_termination()));
```

Comment:
This violates the documentation for this function:
`All strings are null-terminated.`

https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glGetString.xhtml

Reply:
This string is `null` terminated. The function is not applying NULL termination to it. It is confusing a bit because the `null` termination status depends on how the `StringView` was constructed. I think the name of that function should be changed. I don't know why it is what it is.

Anyways, it can be changed to not have this confusion.

Reply:
Changed to be a `char const*` instead of a `StringView`